### ISSUE TITLE
[Request] Multiple AVARS in sequence 

### ISSUE THREAD

**Issue by jbud** [Fragtality/PilotsDeck#39]
Request sending AVARS in sequence for a single button. For example FBW A32NX RWY Turn lights would look like this:

(A:LIGHT TAXI:2, Bool):(A:LIGHT TAXI:3, Bool)

As there is no single target or event for this switch.

**Comment by Fragtality** [comment]
I don't think that will happen for Command & Variable Actions (it's just not A-Vars what you're requesting there).
Also there Alternatives through RPN/Calculator Code or FSUIPC Lua Scripts.

**Comment by jbud** [comment]
Perhaps I am just new at this and don't understand the RPC code to perform this task

**Comment by Fragtality** [comment]
Well, I coded a whole StreamDeck Plugin and still RPN-Code gives me shivers 🤣 
Have you checked my PilotsDeck Profile for the FBW on flightsim.to?

If not, try that for the Rwy Turn -Off Lights:
67072=21:67072:22
(Type CONTROL)

For State:
(A:CIRCUIT SWITCH ON:22, Bool)

**Comment by jbud** [comment]
> Well, I coded a whole StreamDeck Plugin and still RPN-Code gives me shivers 🤣 Have you checked my PilotsDeck Profile for the FBW on flightsim.to?

I have. I thusfar have setup my streamdeck with no LUA and remain on the free version of FSUIPC, else I would have just gone that route.

> If not, try that for the Rwy Turn -Off Lights: 67072=21:67072:22 (Type CONTROL)
> 
> For State: (A:CIRCUIT SWITCH ON:22, Bool)

This did the trick, however unintuitively, I do understand my ask is likely a large change. This does solve my issue so I will close this out.

**Comment by Fragtality** [comment]
The Explanation to that: 
FBW has Documented, the Runway Turn-Off Lights (and Switch *sigh*) can also be toggled via the MSFS Var (aka A-Var, aka SimVar) "CIRCUIT SWITCH ON:21" and "CIRCUIT SWITCH ON:22".
So looking at the available FSUIPC Commands (which are just Numbers to what is also know as K-Var or SimConnect Event) I discovered there is a Command for "ELECTRICAL_CIRCUIT_TOGGLE" (which has the Number 67072).
Therefore I manipulated these A-Vars through a Command (and Command Only Types are chainable).
